Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[workspace.package]
+authors = ["Polkadot Fellowship"]
```

Comment:
I guess this is fine for a start?  
The files still have the parity file header, so maybe we should include parity here as well?